team-telnyx/telnyx-webrtc-android
# [WEBRTC-673] Automatic login if logged in before

[WebRTC-673 - Automatic login.](https://telnyx.atlassian.net/browse/WEBRTC-673)
---

This PR reintroduces auto login which was removed temporarily when the login process changed. 

## :older_man: :baby: Behaviors
### Before changes
Manual login each time

### After changes
If logged in before, and have not manually disconnected, we will automatically login the last user each time the application has launched.

**Note:** gateway registration can still fail, which means a manual login might be required if the automatic login fails. 

## ✋ Manual testing
1. Launch the application, log in manually for the first time
2. Close the application, launch it again. You should be logged in automatically
3. Disconnect manually
4. Launch the application again, you will need to login manually again
